recipes/pgsu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pgsu" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: af33bc917b26cede741f53953f54e5cef9d6428bd56a68f37198ac7879e99e83
+
+build:
+  number: 0
+  entry_points:
+    - pgsu=pgsu.cli:run
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ py < 27 or py > 38 ]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click
+    - enum34;   # [ py < 35 ]
```

Comment:
```suggestion
    - enum34   # [ py < 35 ]
```

Reply:
Actually, you can remove this deps, we just create packages for Python 2.7 and 3.6+